maintenance_equipment_image/tests/test_maintenance_equipment_image.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2021 Exo Software
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+import base64
+import io
+
+from PIL import Image
+
+from odoo.tests import common
+
+
+class TestMaintenanceEquipmentImage(common.TransactionCase):
+    def setUp(self):
+        super(TestMaintenanceEquipmentImage, self).setUp()
+
+        # Create an image
+        f = io.BytesIO()
+        Image.new("RGB", (1920, 1080), "#4169E1").save(f, "PNG")
+        f.seek(0)
+        sample_image = base64.b64encode(f.read())
+
+        self.Equipment = self.env["maintenance.equipment"]
+        self.equipment1 = self.Equipment.create(
+            {
+                "name": "Equipment 1",
+                "image": sample_image,
+            }
+        )
```

Comment:
What do you test in this test??

I think nothing.

Reply:
Well, tests are allways welcome, but they are not correctly executed (some ifs are ignored) and in this case are not really needed. 

Also, make all in a single commit please

Reply:
If in this case are not really needed, then remove tests. Why have something that doesn't work?

Reply:
Thank you for commenting. I removed the tests and squashed the commits into one by force pushing. Can you please check?